### ISSUE TITLE
Mention that Raku can use Unicode literals with the No or Nd characteristic

### ISSUE THREAD

**Issue by JJ** [Raku/doc#3194]
## The problem

Anything that has a ["number-like" Unicode property ](https://en.wikipedia.org/wiki/Unicode_character_property) can be used as such in Raku

## Suggestions

Show what can be done with Nd such as ٨ or No such as ⅓, which as far as I can tell, [is pretty unique to Raku](https://stackoverflow.com/questions/60008149/which-languages-can-do-arithmetics-with-unicode-nd-or-no-codepoints)

**Comment by AlexDaniel** [comment]
Pretty sure it's mentioned on https://docs.raku.org/language/unicode_ascii

**Comment by JJ** [comment]
[That](https://docs.raku.org/language/unicode_ascii#Numeric_values) does not mention Nd, and does not mention they can be used as [literals](https://docs.raku.org/language/syntax#index-entry-Number_(literals)). I'll modify the suggestion to include that.

**Comment by AlexDaniel** [comment]
Just look a little bit higher.

![image](https://user-images.githubusercontent.com/5507503/73558728-6fb47980-445c-11ea-9c11-a2e10158ef3e.png)

**Comment by JJ** [comment]
Again, it's not mentioned in the literal section, but in an adhoc page about Unicode. It can be moved, or enhanced and linked.

**Comment by JJ** [comment]
Mainly what I want is that looking up "unicode numeric literals" or something like that is easy to find for Raku.

**Comment by JJ** [comment]
Also, maybe "Numeric characters" should be changed to numeric *digits*. ⅓ is a character, it's not a digit.

**Comment by treyharris** [comment]
On Fri, Jan 31, 2020 at 12:11 Juan Julián Merelo Guervós <
notifications@github.com> wrote:

> Also, maybe "Numeric characters" should be changed to numeric *digits*. ⅓
> is a character, it's not a digit.
>

Don’t you mean the other way around? I agree ⅓ is a character, not a digit,
so it’s a numeric character, not a numeric digit. Or are you talking about
a different spot?

**Comment by JJ** [comment]
Yep, but that specific section does not include 1/3; the next one does.
